### DISCUSSION TITLE
Can I use Week 7's assets for my project?

### DISCUSSION THREAD

**Discussion by htv04** [FunkinCrew/Funkin#1730] [General]
Hi, I was just wondering if it's okay to use Week 7's assets for my project, [Friday Night Funkin' Rewritten](https://github.com/HTV04/funkin-rewritten).

I know mods usually can't use Week 7's assets, but my rewrite doesn't use any of the original game's code, so I do not view it as a mod. Given your responses to similar questions though, I'm still on the fence about whether it's okay to use the assets or not, so I wanted to get your approval on it first.

**Comment by ninjamuffin99** [comment]
no

**Reply by htv04** [reply]
Okay, I understand. Thanks anyway.
